### PR TITLE
LPS-74373 Document deprecation of portal.properties

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

**Comment by izaera** [comment]
Hey @codyhoag:

Can you rephrase the portal.properties documentation as requested by Brian (see https://github.com/brianchandotcom/liferay-portal/commit/d30fc307bb9a800b0f6663eb46832de4d7f4e195)? 

Thx

**Comment by izaera** [comment]
ci:stop

**Comment by izaera** [comment]
Stopped CI because this PR has no executable code.

**Comment by codyhoag** [comment]
Sent brianchandotcom#51564
